otherlibs/stdune/src/path.ml

```diff
@@ -505,7 +505,7 @@ end = struct
     if prefix = path
     then Some Local.root
     else (
-      let prefix = prefix ^ "/" in
+      let prefix = if String.is_suffix ~suffix:"/" prefix then prefix else prefix ^ "/" in
```

Comment:
Can you make sure the `drop_prefix` works in a consistent way for non external paths as well? I think it might rely on the fact that `Path.Local.t` never contains a leading `/`, but it's good to double check.

Reply:
I checked and this was failing, so I pushed a fix for this as well